backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/card/BpkCard.kt

```diff
@@ -0,0 +1,74 @@
+package net.skyscanner.backpack.compose.card
+
+import androidx.compose.animation.animateColorAsState
+import androidx.compose.animation.core.animateDpAsState
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.interaction.collectIsFocusedAsState
+import androidx.compose.foundation.interaction.collectIsPressedAsState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.Role
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkBorderRadius
+import net.skyscanner.backpack.compose.tokens.BpkElevation
+
+enum class BpkCardCorner {
+  Small,
+  Large,
+}
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun BpkCard(
```

Comment:
the previous card had a padded vs not behaviour to keep the padding consistent to 16dp - is it intentional to remove that?

Reply:
The padding is not a part of the Firma design – and I think there could be some cases when padding will not be needed for the card content, so it should not be a property.

Reply:
currently all platforms have a padded & non-padded version, so this would be consistent with the rest of the implementations